dix/req.c

```diff
@@ -1629,11 +1629,23 @@ static void dix_rop_completed(struct m0_sm_group *grp, struct m0_sm_ast *ast)
 	(void)grp;
 	if (req->dr_type == DIX_NEXT)
 		m0_dix_next_result_prepare(req);
-	else
-		m0_tl_for(cas_rop, &rop->dg_cas_reqs, cas_rop) {
-			dix_cas_rop_rc_update(cas_rop, 0);
+	else {
+		/*
+		 * Return success if there is atleast one successful CAS
```

Comment:
Let's mention DIX here, split "atleast" and add comma at the end.

"Consider DIX request to be successful if there is at least one successful CAS request."